Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -105,340 +78,245 @@ final class KSRAnalyticsTests: TestCase {
 
-      ksrAnalytics.trackTabBarClicked(.activity)
+      ksrAnalytics.trackTabBarClicked(.discovery)
 
-      let dataLakeClientProperties = dataLakeClient.properties.last
-      let segmentClientProperties = segmentClient.properties.last
-
-      XCTAssertEqual("es", dataLakeClientProperties?["session_display_language"] as? String)
-      XCTAssertEqual("es", segmentClientProperties?["session_display_language"] as? String)
+      XCTAssertEqual("es", segmentClient.properties.last?["session_display_language"] as? String)
     }
   }
 
   func testSessionProperties_VoiceOver() {
-    let dataLakeClient = MockTrackingClient()
     let segmentClient = MockTrackingClient()
-    let ksrAnalytics = KSRAnalytics(dataLakeClient: dataLakeClient, segmentClient: segmentClient)
+    let ksrAnalytics = KSRAnalytics(segmentClient: segmentClient)
 
     withEnvironment(isVoiceOverRunning: { true }) {
-      ksrAnalytics.trackTabBarClicked(.activity)
+      ksrAnalytics.trackTabBarClicked(.discovery)
 
-      let dataLakeClientProperties = dataLakeClient.properties.last
       let segmentClientProperties = segmentClient.properties.last
 
-      XCTAssertEqual(true, dataLakeClientProperties?["session_is_voiceover_running"] as? Bool)
       XCTAssertEqual(true, segmentClientProperties?["session_is_voiceover_running"] as? Bool)
     }
 
     withEnvironment(isVoiceOverRunning: { false }) {
-      ksrAnalytics.trackTabBarClicked(.activity)
+      ksrAnalytics.trackTabBarClicked(.discovery)
 
-      let dataLakeClientProperties = dataLakeClient.properties.last
       let segmentClientProperties = segmentClient.properties.last
 
-      XCTAssertEqual(false, dataLakeClientProperties?["session_is_voiceover_running"] as? Bool)
       XCTAssertEqual(false, segmentClientProperties?["session_is_voiceover_running"] as? Bool)
     }
   }
 
   func testSessionProperties_LoggedIn() {
-    let dataLakeClient = MockTrackingClient()
     let segmentClient = MockTrackingClient()
     let ksrAnalytics = KSRAnalytics(
-      dataLakeClient: dataLakeClient,
       loggedInUser: User.template,
       segmentClient: segmentClient
     )
 
-    ksrAnalytics.trackTabBarClicked(.activity)
+    ksrAnalytics.trackTabBarClicked(.discovery)
 
-    let dataLakeClientProperties = dataLakeClient.properties.last
     let segmentClientProperties = segmentClient.properties.last
 
-    XCTAssertEqual(true, dataLakeClientProperties?["session_user_is_logged_in"] as? Bool)
     XCTAssertEqual(true, segmentClientProperties?["session_user_is_logged_in"] as? Bool)
   }
 
   func testSessionProperties_DeviceFormatAndClientPlatform_ForIPhoneIdiom() {
-    let dataLakeClient = MockTrackingClient()
     let segmentClient = MockTrackingClient()
     let ksrAnalytics = KSRAnalytics(
-      dataLakeClient: dataLakeClient,
       device: MockDevice(userInterfaceIdiom: .phone),
       loggedInUser: nil,
       segmentClient: segmentClient
     )
-    ksrAnalytics.trackTabBarClicked(.activity)
```

Comment:
Because after removing DataLake we are only track` .discovery` tab in `trackTabBarClicked` for now. Keeping `.activity ` will result in empty bracket assert `[]`.